pygin/pygin/src/helpers.hpp

```diff
@@ -45,4 +46,30 @@ namespace helpers
 			return Result;
 		}
 	}
+
+	class py_string_storage
+	{
+	public:
+		py_string_storage() {}
+
+		py_string_storage(py_string_storage const&) = delete;
+		py_string_storage& operator= (py_string_storage const&) = delete;
+
+		~py_string_storage(){ this->clear(); }
```

Comment:
This shouldn't be needed with a proper resource management.

Reply:
I'd strongly prefer *any* number of containers to manual release & delete.
Anyways, I can address that separately.